src/sdam/topology_description.ts

```diff
@@ -438,7 +438,7 @@ function updateRsWithPrimaryFromMember(
 ): TopologyType {
   if (setName == null) {
     // TODO(NODE-3483): should be an appropriate runtime error
-    throw new MongoDriverError('Argument "setName" is required if connected to a replica set');
+    throw new MongoRuntimeError('Argument "setName" is required if connected to a replica set');
```

Comment:
is setName user specified? if not, this would be a runtime error

Reply:
It seems that `setName` is created when `TopologyDescription` is instantiated and that class is exported in src/index.ts, so I'll go ahead and change this to `MongoAPIError`

Reply:
huh, I think I was originally reading it as an API error, but I'm pretty sure it is a runtime error, which it currently is in this code, so no need to change it